docs/index.md

```diff
@@ -80,7 +80,8 @@ are summarized below:
     The [syntax rules](http://daringfireball.net/projects/markdown/syntax#list)
     clearly state that when a list item consists of multiple paragraphs, "each
     subsequent paragraph in a list item **must** be indented by either 4 spaces
-    or one tab" (emphasis added). However, many implementations do not enforce
+    or one tab" (emphasis added). However, many implementations – including the
+    original markdown.pl implementation itself – do not enforce
```

Comment:
#125, #313, #442 and #751 mention the original implementation as a reason for Python-Markdown to also implement this, so I figured it'd be helpful to mention that we already know that markdown.pl already behaves the way it does to avoid duplicate issues.

Reply:
People should be searching old issues before opening a new issue, in which case they would know that. I realize GitHub doesn't search closed issues by default, but we call this out in our (new) [contributing guide][1]. If people follow the guide, then this is a non-issue.

[1]: https://github.com/Python-Markdown/markdown/blob/master/docs/contributing.md#issues

Reply:
People don't. I know, because I didn't (until afterwards).